scripts/generator/graphql_generator/csharp/SDK/iOS/iOSNativeCheckout.ApplePayEventReceiver.cs.erb

```diff
@@ -60,7 +60,76 @@ namespace <%= namespace %>.SDK.iOS {
         }
 
         public void FetchApplePayCheckoutStatusForToken(string serializedMessage) {
```

Comment:
I'm not sure about the name of this method. It's named `Fetch..` but doesn't return anything. It looks like it finishes by sending a the response back to the native side. Is it possible to name it to reflect that?